website/ui/src/app/models/events.ts

```diff
@@ -132,3 +141,17 @@ export interface Purchase extends EcommerceObject {
   shipping?: number;
   tax?: number;
 }
+
+export interface Login extends Event {
```

Comment:
Do you need two interfaces here?  Could you not cover both under an `AuthEvent` and have the logged_in attribute as a boolean?